README.md

```diff
@@ -1 +1,6 @@
-# oss-cloud
\ No newline at end of file
+# oss-cloud
+
+To run the project, a file named ".env" i required in the oss-cloud-backend directory.

```

Comment:
Do you create .env file in local by yourself?

Take a look at serverless env variables, serverless create .env file for you and manage variables that you define within serverless.yml file.